packages/typespec-client-generator-core/src/internal-utils.ts

```diff
@@ -546,3 +553,38 @@ export function isXmlContentType(contentType: string): boolean {
   const regex = new RegExp(/^(application|text)\/(.+\+)?xml$/);
   return regex.test(contentType);
 }
+
+/**
+ * If body is from spread, then it does not directly from a model property.
+ * @param httpBody
+ * @param parameters
+ * @returns
+ */
+export function isHttpBodySpread(httpBody: HttpOperationBody | HttpOperationMultipartBody) {
+  return httpBody.property === undefined;
+}
+
+/**
+ * If body is from simple spread, then we use the original model as body model.
+ * @param type
+ * @returns
+ */
+export function getHttpBodySpreadModel(type: Model): Model {
+  if (type.sourceModels.length === 1 && type.sourceModels[0].usage === "spread") {
+    const innerModel = type.sourceModels[0].model;
+    // for case: `op test(...Model):void;`
+    if (innerModel.name !== "" && innerModel.properties.size === type.properties.size) {
+      return innerModel;
+    }
```

Comment:
Did we consider the case that the `Model` here include `@path` `@query` etc.?

Reply:
nope, see pr description for details.

Reply:
Wondering which line exclude this scenario. `innerModel.properties.size === type.properties.size`?

If `Model` contains `@query`, would the size be different?